.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+
+cache:
+    pip: true
+    directories:
+    - $HOME/Library/Caches/Homebrew
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      python: "3.5"
+    - os: linux
+      dist: trusty
+      python: "3.6"
+    - os: osx
+      osx_image: xcode8.3
+      sudo: required
+      language: generic
+      before_install:
+      #  Travis bug: manually install python on osx
+      #  See https://github.com/travis-ci/travis-ci/issues/2312#issuecomment-289140791
```

Comment:
@rht I will have a look through this issue thread as I don't understand the reason for the complexity of testing Python 3.6 on MacOSX.  Perhaps something to do with PyPy dependency?